kubeportal/api/views.py

```diff
@@ -70,7 +66,7 @@ class WebApplicationViewSet(viewsets.ReadOnlyModelViewSet):
     serializer_class = WebApplicationSerializer
 
     def get_queryset(self):
-        if 'user_pk' in self.kwargs:
+        if 'user_pk' in self.kwargs and self.kwargs['user_pk'] != 'null':
             # Query for webapp list of a specific user
             # For the moment, a user can only query its own web apps.
             query_pk = int(self.kwargs['user_pk'])
```

Comment:
Please wrap this in try/except and react on broken parameters with a proper HTTP response + log line.